### PR TITLE
Added examples and other small variations to docstrings

### PR DESCRIPTION
**Pull request by MatteoGallet**

### CODE REVIEW

File: pyrigi/framework.py [hunk 1]
```diff
@@ -258,14 +298,21 @@ def from_graph(cls, graph: Graph, dim: int) -> None:
 
         N = 10 * len(graph.vertices())**2 * dim
         realization = {
-                vertex: [randrange(1,N) for _ in range(dim)] 
+            vertex: [randrange(1, N) for _ in range(dim)]
```
Comment (matthiashimmelmann) [inline pyrigi/framework.py:301]: I find this kind of notation really hard to read in-code, even if it is PEP8. Maybe we can agree upon a different notation that is also PEP8-conforming?

File: pyrigi/framework.py [hunk 2]
```diff
@@ -543,21 +637,26 @@ def stress_matrix(
         """
         raise NotImplementedError()
 
-    def trivial_infinitesimal_flexes(
-            self, pinned_vertices: Dict[Vertex, List[int]] = {}) -> List[Matrix]:
+    def trivial_infinitesimal_flexes(self,
+                                     pinned_vertices: Dict[Vertex,
+                                                           List[int]] = {}) -> List[Matrix]:
```
Comment (matthiashimmelmann) [inline pyrigi/framework.py:642]: This segmentation is super weird.

**Review by matthiashimmelmann — APPROVED** [review]

File: pyrigi/framework.py (hunk 1 above)
Comment (Legersky) [reply pyrigi/framework.py:287]: [black](https://github.com/psf/black) formats it like this:
```python
        realization = {
            vertex: [randrange(1, N) for _ in range(dim)] for vertex in graph.vertices()
```

File: pyrigi/framework.py (hunk 2 above)
Comment (MatteoGallet) [reply pyrigi/framework.py:601]: I agree, I will improve it in a PEP8-complaint way in a next feature branch.

Comment (Legersky) [reply pyrigi/framework.py:601]: black:
```python
    def trivial_infinitesimal_flexes(
        self, pinned_vertices: Dict[Vertex, List[int]] = {}
    ) -> List[Matrix]:
```

**Pull request merged**